src/migrate/Migrator.js

```diff
@@ -49,12 +49,16 @@ export default class Migrator {
   constructor(knex) {
     // Clone knex instance and remove post-processing that is unnecessary for internal queries from a cloned config
     if (isFunction(knex)) {
-      this.knex = knex.withUserParams({
-        ...knex.userParams,
-      });
-      this.knex.disableProcessing();
```

Comment:
Moved this inside operations instead to ensure correct state before each of them, regardless of how previous ones ended.